<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>转换工具</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresource.qrc">
    <normaloff>:/images/calculate.ico</normaloff>:/images/calculate.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QComboBox" name="comboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>十六进制转换成十进制</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>十进制转换成十六进制</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>十六进制转换成二进制</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>二进制转换成十六进制</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ASCII转换成十六进制</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>十六进制转换成ASCII</string>
               </property>
              </item>
             </widget>
             <widget class="QPushButton" name="pBtn_Clear">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>清除</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>转换</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPlainTextEdit" name="plainTextEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QComboBox" name="comboBox_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>十六进制转换成十进制</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>十进制转换成十六进制</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>十六进制转换成二进制</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>二进制转换成十六进制</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ASCII转换成十六进制</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>十六进制转换成ASCII</string>
               </property>
              </item>
             </widget>
             <widget class="QPushButton" name="pBtn_Clear2">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>清除</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>转换</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="myresource.qrc"/>
 </resources>
 <connections/>
</ui>
